components/basecomponents/Exercise/components/ExerciseIdentificationComponent.tsx

```diff
@@ -106,14 +105,17 @@ export const ExerciseIdentificationComponent = forwardRef(({ exercise }: Exercis
             : t('exercise_page.exercise_text_idenfification_hint')}
         </p>
         <div className="flex w-full items-center justify-center">
-          <h5 ref={mainTextRef} className={`text-xl sm:text-2xl text-center p-0 ${mode === 'audio' ? 'opacity-0' : ''}`}>
-            {exercise.getText()}
-          </h5>
-          <div ref={soundwaveRef} className={`absolute w-40 inline ${mode === 'text' ? 'opacity-0' : ''}`}>
-            <SoundWaveIcon className="inline h-auto" />
-          </div>
+          {mode === 'text' && (
```

Comment:
After this modification text won't appear after exercise completion. Text and soundwave icon must be rendered both because of content jumping.

Reply:
Yeah, but the SVG interferes with being able to select the text :/ I will create a ticket to address this separately

Reply:
can be done with z-index